e2e/testcases/helm_sync_test.go

```diff
@@ -328,3 +337,15 @@ func helmChartVersion(chartVersion string) nomostest.Sha1Func {
 		return chartVersion, nil
 	}
 }
+
+func deleteWordpressPVC(nt *nomostest.NT) error {
```

Comment:
Is this created indirectly? Why doesn't the normal cleanup delete it? Are we missing Cleanup that removes the helm chart and deletes all the bits in the inventory?

Reply:
It is not an object declared in the chart and managed by Config Sync. I believe MariaDB creates it on the cluster to persist data, but never deletes it even when MariaDB is deleted.

Reply:
Is this a generic problem users will have? Do we need to document it?

How is the PVC created? Is it the result of a Deployment field?

Is there a way to configure the chart not to use persistent storage?

Reply:
If user's workload creates new K8s resources and doesn't have a finalizer to remove them, then it is a problem.
The PVC seems to be created by MariaDB, which is not a field specified in the Deployment.
Let me check the chart values to see whether we can disable persistence.

Reply:
Done. Updated the chart value to disable mariadb persistent storage.

Reply:
New config looks good!